sagemaker-pipeline-parameterization/notebook.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    # Use the sklearn_processor in a Sagemaker pipelines ProcessingStep
</pre><p>Grammar/branding - change all instances of <code>Sagemaker</code> to <code>SageMaker</code> ﻿throughout the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3021/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>